pyupgrade.py

```diff
@@ -1122,6 +1122,9 @@ def _fix_percent_format(contents_text: str) -> str:
     'assertRaisesRegex': '{args[0]}.assertRaisesRegex({rest})',
     'assertRegex': '{args[0]}.assertRegex({rest})',
 }
+SIX_CALLS_PARENTHESIZED = {
+    k: v.replace('{args[0]}', '({args[0]})') for k, v in SIX_CALLS.items()
+}
```

Comment:
this isn't quite correct, some of the calls in `SIX_CALLS` do not need this treatment (for example `int2byte`)

Reply:
Well, of course. My arguments against it were code duplication, no effect where it does not apply, prevention of missing entries, lower maintenance. Due to the stable state of `six` in reality I will not fight for it now, but I will come back on this once Python4 comes around and `six`will experience a revival ;-)

Reply:
you don't need any duplicate information, the parenthesizing can be done as an option to the formatter

Reply:
Uh - and also I have no clue for which one it applies really (beside my specific bug report and your mentioned one) ...

Reply:
ah ok, it's pretty easy to see from the mapping there -- imagine putting `1 + 2` for `args[0]` and see which ones would be syntactically different than `(1 + 2)` for `args[0]` and then you know which ones to specialize